docs/contributing.md

```diff
@@ -38,7 +38,13 @@ use `jj split` to do it if you didn't realize ahead of time how it should be
 split up. Include tests and documentation in the same commit as the code they
 test and document. The commit message should describe the changes in the commit;
 the PR description can even be empty, but feel free to include a personal
-message.
+message. We write commit messages in an affected component style and don't use 
```

Comment:
This paragraph is getting long. Maybe break before "The commit message should ..."?

Reply:
added a break.